torch/csrc/distributed/c10d/Functional.cpp

```diff
@@ -6,6 +6,7 @@
 #include <torch/csrc/distributed/c10d/GroupRegistry.hpp>
 #include <torch/csrc/distributed/c10d/ProcessGroup.hpp>
 #include <torch/csrc/distributed/c10d/RankLocal.hpp>
+#include <iostream>
```

Comment:
this was debugging? need to remove this I think

Reply:
sorry, good catch